### PR TITLE
Add missing standards, organize standards, and update links in source README

### PR DESCRIPTION
**Pull request by bitzoic**
## Type of change

- Improvement (refactoring, restructuring repository, cleaning tech debt, ...)
- Documentation

## Changes

The following changes have been made:

- The SRC-6 and SRC-8 standards were missing from the source README
- The links to the standards in the source README were broken and out of date
- The standards have been sorted and categorized for easier navigation
- The examples for the SRC-6 standard were missing from the source README
- The link to Native Assets documentation for the SRC-7 standard in the source README was out of date

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,26 @@ If you don't find what you're looking for, feel free to create an issue and prop
...
-- [SRC-2; Inline Documentation](./standards/src_2/) defines how to document your Sway files.
-- [SRC-3; Mint and Burn](./standards/src_3/) is used to enable mint and burn functionality for Native Assets.
-- [SRC-5; Ownership Standard](./standards/src_5/) is used to restrict function calls to admin users in contracts.
-- [SRC-7; Arbitrary Asset Metadata Standard](./standards/src_7/) is used to store metadata for [Native Assets](https://docs.fuel.network/docs/sway/blockchain-development/native_assets).
-- [SRC-9; Metadata Keys Standard](./standards/src_9/) is used to store standardized metadata keys for [Native Assets](https://fuellabs.github.io/sway/v0.44.0/book/blockchain-development/native_assets.html) in combination with the SRC-7 standard.
+### Native Assets
+
+- [SRC-20; Token Standard](./standards/src20-token/) defines the implementation of a standard API for [Native Assets](https://docs.fuel.network/docs/sway/blockchain-development/native_assets) using the Sway Language.
```
Comment (K1-R1) [inline README.md:38]: Would be nice to have these in numerical order, will be helpful as the lists continue to grow

File: README.md [hunk 2]
```diff
@@ -98,6 +112,20 @@ Example of the SRC-5 implementation where a contract does not have an owner set
 
 Example of the SRC-5 implementation where a contract has an owner set at compile time.
 
+#### SRC-6; Vault Standard Examples
+
+##### [Multi Token Vault](./examples/src6-vault/multi_token_vault/)
+
+A barebones implementation of the vault standard that supports any number of sub vaults being created for every AssetId.
```
Comment (K1-R1) [inline README.md:119]:
```suggestion
A basic implementation of the vault standard that supports any number of sub vaults being created for every AssetId.
```

File: README.md [hunk 3]
```diff
@@ -98,6 +112,20 @@ Example of the SRC-5 implementation where a contract does not have an owner set
...
+
+##### [Single Token Vault](./examples/src6-vault/single_token_vault/)
+
+A barebones implemenation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId.
```
Comment (K1-R1) [inline README.md:123]:
```suggestion
A basic implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId.
```

File: README.md [hunk 4]
```diff
@@ -98,6 +112,20 @@ Example of the SRC-5 implementation where a contract does not have an owner set
...
+
+##### [Single Token Single Sub Vault](./examples/src6-vault/single_token_single_sub_vault/)
+
+A barebones implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId, and to a single Sub vault.
```
Comment (K1-R1) [inline README.md:127]:
```suggestion
A basic implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId, and to a single Sub vault.
```

File: README.md (hunk 1 above)
Comment (bitzoic) [reply README.md:38]: I agree but in this case I do feel the token standard should go before any of the token extensions. We originally named it SRC-20 in homage to ERC-20 but in hindsight that might cause problems

**Review by K1-R1 — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -98,6 +112,20 @@ Example of the SRC-5 implementation where a contract does not have an owner set
...
 
+#### SRC-6; Vault Standard Examples
+
+##### [Multi Token Vault](./examples/src6-vault/multi_token_vault/)
+
+A basic implementation of the vault standard that supports any number of sub vaults being created for every AssetId.
+
+##### [Single Token Vault](./examples/src6-vault/single_token_vault/)
+
+A basic implemenation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId.
```
Comment (Braqzen) [inline README.md:123]:
```suggestion
A basic implementation of the vault standard demonstrating how to restrict deposits and withdrawals to a single AssetId.
```

File: README.md [hunk 6]
```diff
@@ -98,6 +112,20 @@ Example of the SRC-5 implementation where a contract does not have an owner set
...
+
+##### [Single Token Single Sub Vault](./examples/src6-vault/single_token_single_sub_vault/)
+
+A basic implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId, and to a single Sub vault.
```
Comment (Braqzen) [inline README.md:127]:
```suggestion
A basic implementation of the vault standard demonstrating how to restrict deposits and withdrawals to a single AssetId, and to a single Sub vault.
```

**Review by Braqzen — APPROVED** [review]

**Review by SwayStar123 — APPROVED** [review]

**Pull request merged**